src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -963,6 +963,19 @@ ov::runtime::Tensor generate(const
     return tensor;
 }
 
+ov::runtime::Tensor generate(const
+                             std::shared_ptr<ov::op::v8::DeformableConvolution>& node,
+                             size_t port,
+                             const ov::element::Type& elemType,
+                             const ov::Shape& targetShape) {
+    if (1 == port) {
```

Comment:
Could you lease use `InputGenerateData` structure as by default way, because we can use set values by conformance

Reply:
done